tests/test_client.py

```diff
@@ -235,7 +235,7 @@ def create(self, stream, start_date, end_date=None, num_records=1): # pylint: di
         else:
             raise NotImplementedError("create not implemented for stream {}".format(stream))
 
-    def get_first_found(self, stream, start_date):
+    def get_or_create_first_found(self, stream, start_date):
```

Comment:
This rename was from a previous PR feedback requested by @kspeer825  so I carried it over here